### PR TITLE
String First task done

### PR DESCRIPTION
**Pull request by adushyn**

### CODE REVIEW

File: src/strings/practice/1.js [hunk 1]
```diff
@@ -6,13 +6,34 @@
 
 module.exports = function capFirstLowLast(str) {
 	'use strict';
-	// your code
+	let firstChar = str[0];
+	let firstCharUpper = firstChar.toUpperCase();
```
Comment (ODushyn) [inline src/strings/practice/1.js]: you may change those two lines with one: `let firstCharUpper = str[0].toUpperCase();`

File: src/strings/practice/1.js [hunk 2]
```diff
@@ -6,13 +6,34 @@
...
+
+	let lastChar = str[str.length - 1];
+	let lastCharLower = lastChar.toLowerCase();
```
Comment (ODushyn) [inline src/strings/practice/1.js]: same here: `let lastCharLower = str[str.length - 1].toUpperCase();`

File: src/strings/practice/1.js (hunk 1 above)
Comment (adushyn) [reply src/strings/practice/1.js]: done

File: src/strings/practice/1.js (hunk 2 above)
Comment (adushyn) [reply src/strings/practice/1.js]: done

**Pull request merged**